src/assets/style.css

```diff
@@ -61,7 +61,6 @@
   width: 100%;
   height: 100%;
   position: absolute;
-  visibility: hidden;
   z-index: -1;
```

Comment:
let's remove the border as well, and set bigger negative z-index for safety.
Good catch!